Manifest.toml

```diff
@@ -47,7 +47,7 @@ deps = ["Libdl"]
 uuid = "deac9b47-8bc7-5906-a0fe-35ac56dc84c0"
 
 [[LibGit2]]
-deps = ["Printf"]
+deps = ["Base64", "NetworkOptions", "Printf", "SHA"]
```

Comment:
I don't think the Julia we use here likes `NetworkOptions`

Reply:
The Manifest was modified using Julia 1.6 (specifically commit a178ac2cba) so this should be good.